lib/media/protocol/db_protocols.ex

```diff
@@ -8,7 +8,7 @@ defprotocol DB do
   def delete_media(struct)
   def count_namespace(struct)
   def content_medias(struct)
-
+  def dereference_content(struct)
```

Comment:
I couldn't find the PGSQL implementation of this function

Reply:
yes there is none but just to raise your attention to the fact that in case of a postgresql it is a joint table if I remember correctly as there is a many to many relationship between these two.

Reply:
but then if the context module calls the DB.dereference_content function on a PgSQL what will happen ?

Reply:
yes it would crash, what do you think I should do here? removed the entries manually from the joint table ?

Reply:
update the doc. to state that it's not supported yet with a link to the issue in relation with that

Reply:
done